extensions/lifx/src/turnOffAllLights.tsx

```diff
@@ -22,7 +22,7 @@ export default async function TurnOffAllLights() {
       return;
     }
     const response = await SetLightState("all", body, config);
-    await showHUD("Succses");
+    await showHUD("Success");
```

Comment:
Would it make sense to give this a better message?
```suggestion
    await showHUD("Turned off all lights");
```

Reply:
Yup that makes sense. 👍